game-core/src/main/java/games/strategy/net/ClientMessenger.java

```diff
@@ -212,8 +228,9 @@ public boolean isServer() {
 
   @Override
   public void socketUnqaurantined(
-      final SocketChannel channel, final QuarantineConversation converstaion2) {
-    final ClientQuarantineConversation conversation = (ClientQuarantineConversation) converstaion2;
+      final SocketChannel channel, final QuarantineConversation quarantineConversation) {
+    final ClientQuarantineConversation conversation =
+        (ClientQuarantineConversation) quarantineConversation;
```

Comment:
I feel like this might be a good use case for the var keyword

Reply:
Sure: 3fb3d8e8a
The real change here is to fix a spelling error in the parameter. That probably should have been done in a different PR. It's not good practice to throw in random refactorings when adding a feature, particularly when deeper and more meaningful fixes/refactorings are more called for to really fix some of the problems we have.

While fixing spelling, the method name is misspelled, fixed in: c5c340d752c9a76e9d1fa372183378655bdf18f9